src/sdgio-edit.owl

```diff
@@ -1596,6 +1601,15 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020237> "sub-outcome of an intermediate development outcome"@en)
 SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00000251>)
 
+# Class: <http://purl.unep.org/sdg/SDGIO_00020300> (cross-cutting objective)
```

Comment:
This should be "CGIAR cross-cutting objective"

Reply:
I fixed. I wonder whether, for uniformity, also the other classes in the same superclass should being with 'CGIAR'. So, for example, should I change 'System level outcome' to 'CGIAR system level outcome'? 

I guess we first did not do that because the superclass already makes it clear that we are talking about CGIAR. The superclass is 'CGIAR objective specification'.

Reply:
> So, for example, should I change 'System level outcome' to 'CGIAR system level outcome'?

It may make sense, but I think the idea of a "system level outcome" is quite specific. Make an issue about it and we'll consider it after we get more content in.

Reply:
I created issue #165 to take care of this problem.